code/modules/mob/living/simple_animal/hostile/winter_mobs.dm

```diff
@@ -0,0 +1,147 @@
+//////////////////////////
+//		Winter Mobs		//
+//////////////////////////
+
+/mob/living/simple_animal/hostile/winter
+	faction = list("hostile", "syndicate", "winter")
+	speak_chance = 0
+	turns_per_move = 5
+	speed = 1
+	maxHealth = 50
+	health = 50
+	icon = 'icons/mob/winter_mob.dmi'
+	icon_state = "placeholder"
+	icon_living = "placeholder"
+	icon_dead = "placeholder"
+
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	minbodytemp = 0
+
+	melee_damage_lower = 3
+	melee_damage_upper = 7
+
+	var/weapon1
+
+/mob/living/simple_animal/hostile/winter/snowman
+	name = "snowman"
+	desc = "A very angry snowman. Doesn't look like it wants to play around..."
+	icon_state = "snowman"
+	icon_living = "snowman"
+	icon_dead = "snowman-dead"
+	weapon1 = /obj/item/weapon/melee/candy_sword
+
+	bodytemperature = 73.0		//it's made of snow and hatred, so it's pretty cold.
+	maxbodytemp = 280.15		//at roughly 7 C, these will start melting (dying) from the warmth. Mind over matter or something.
+	heat_damage_per_tick = 10	//Now With Rapid Thawing Action!
+
+
+/mob/living/simple_animal/hostile/winter/snowman/death()
+	if(prob(20))	//chance to become a stationary snowman structure instead of a corpse
+		new /obj/structure/snowman(get_turf(src))
+		visible_message("<span class='notice'>The [src.name] shimmers as its animating magic fades away!</span>")
+		..()		//this is just to make sure it gets properly killed before we qdel it
+		qdel(src)
+	else
+		..()
+
+/mob/living/simple_animal/hostile/winter/snowman/ranged
+	ranged = 1
+	retreat_distance = 5
+	minimum_distance = 5
+	projectiletype = /obj/item/projectile/snowball
+	weapon1 = null
+
+/mob/living/simple_animal/hostile/winter/reindeer
+	name = "reindeer"
+	desc = "Apparently murder is a reindeer game."
+	icon_state = "reindeer"
+	icon_living = "reindeer"
+	icon_dead = "reindeer-dead"
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat
+	meat_amount = 3
+
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+
+/mob/living/simple_animal/hostile/winter/santa
+	maxHealth = 150		//if this seems low for a "boss", it's because you have to fight him multiple times, with him fully healing between stages
+	health = 150
+	var/next_stage = null
+	name = "Santa Claus"
+
+	icon_state = "santa"
+	icon_living = "santa"
+	icon_dead = "santa-dead"
+
+/mob/living/simple_animal/hostile/winter/santa/stage_1		//stage 1: slow melee
+	desc = "GET THE FAT MAN!"
+	next_stage = /mob/living/simple_animal/hostile/winter/santa/stage_2
+	speed = 2
+	melee_damage_lower = 5
+	melee_damage_upper = 15
+
+/mob/living/simple_animal/hostile/winter/santa/stage_1/death()
+	..()
+	visible_message("<span class='danger'>HO HO HO! YOU THOUGHT IT WOULD BE THIS EASY?!?</span>")
+	spawn(5)
+		new next_stage(get_turf(src))
```

Comment:
The next_stage bits were actually leftover from a scrapped attempt to handle if someone spawned a base-type Santa mob. I forgot to completely wipe out the bar though, which was my bad.

I'll try and clean that up slightly before family dinner